src/category_theory/monoidal/rigid/basic.lean

```diff
@@ -372,8 +372,12 @@ adjunction.mk_of_hom_equiv
   hom_equiv_naturality_right' :=
     λ X Z Z' f g, tensor_right_hom_equiv_naturality f g, }
 
+/--
+If `Y` has a left dual `ᘁY`, then it is a closed object, with the internal hom functor `Y ⟶[C] -`
+given by left tensoring by `ᘁY`.
+-/
 @[priority 100]
-instance closed_of_has_left_dual (Y : C) [has_left_dual Y] : closed Y :=
+def closed_of_has_left_dual (Y : C) [has_left_dual Y] : closed Y :=
```

Comment:
Can you please include a comment above the docstring explaining which diamond you avoid by making this a def?